src/pages/Home/HomePage.tsx

```
@@ -1,42 +1,69 @@
-            name: '이노인',
-            todayStatus: { takenCount: 3, totalCount: 3 },
-            missedMedications: [],
-            statusMessage: '오늘의 약을 다 복용하였습니다.',
-          },
-        ];
+          missedMedications: [], // 임시: 실제 데이터는 별도 API에서 가져와야 함
+          statusMessage: undefined, // 임시: 실제 데이터는 별도 API에서 가져와야 함
+        }));
+
+        setRecipients(convertedRecipients);
+      } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)